tasks/configuration.yml

```diff
@@ -45,3 +46,7 @@
     backup=yes
   notify: restart rabbitmq-server
   when: rabbitmq_conf_env is defined
+
+- name: set ulimit
+  template: src=rabbitmq_default.j2 dest=/etc/default/rabbitmq-server owner=rabbitmq group=rabbitmq mode=0755
```

Comment:
Can you make this multi-line instead of everything on only one line? It really improve readability.